netbox/dcim/svg/cables.py

```diff
@@ -166,6 +166,8 @@ def _get_color(cls, instance):
         """
         if hasattr(instance, 'parent_object'):
             # Termination
+            if(hasattr(instance, 'get_color') and instance.get_color()):
+                return instance.get_color()
             return 'f0f0f0'
```

Comment:
No need for a method here. You can just do:
```suggestion
            return getattr(instance, 'color', 'f0f0f0')
```

Reply:
This is much cleaner and I like the syntax a lot better, but what I'm finding is that the model has `blank=True` set for both the `FrontPort` and `RearPort` models and this leads to an empty string being stored in the database. This logic fails because it always has the `color` attribute, it just happens to be a blank string.

A quick solution for this would be change that one line to something like below, but I'm open to suggestions.

```suggestion
            return getattr(instance, 'color', 'f0f0f0') or 'f0f0f0'
```

Reply:
That works.